restservice/src/main/java/org/itevents/controller/UserRestController.java

```diff
@@ -85,4 +85,11 @@ public void deactivateSubscription() {
         User user = userService.getUser(userId);
         return eventService.getEventsByUser(user);
     }
+
+    @RequestMapping(method = RequestMethod.POST, value = "/activate/{otp}")
```

Comment:
Why do we use `POST` method? When we go to this link from email the mehod will be `GET`.